src/ModalWrapper.vue

```diff
@@ -33,10 +33,9 @@ export default {
             .map(mapComponent(h, { $style, parent }))
             .filter(a => a);
 
-
         const getIndex = ({ key }) => parent.$modals.active.indexOf(key)
 
-        allModals.sort((a,b) => getIndex(a.data) - getIndex(b.data)).map(a => a.data.key)
+        allModals.sort((a,b) => getIndex(a.data) - getIndex(b.data))
```

Comment:
you dont like semi colon.. :P

Reply:
ah jeez.... you got me

Reply:
my next PR has eslint, so is full semicolon'd